master/pkg/searcher/asha.go

```diff
@@ -272,6 +272,8 @@ func (s *asyncHalvingSearch) trialExitedEarly(
 		s.InvalidTrials++
 		// Remove metrics associated with InvalidHP trial across all rungs
 		highestRungIndex := s.TrialRungs[requestID]
+		rung := s.Rungs[highestRungIndex]
```

Comment:
Why is this in this PR?  I don't understand how it is related.

Reply:
This was a bug in the searcher code where some trials weren't being closed out properly, and since it's only a two line change figured I would merge it in here.

Reply:
Since we are bypassing the `promoteAsync` function due to an InvalidHP, we need to decrement the oustanding trials in a rung normally done by that function, so I do that here.